packages/instant/src/components/scaling_input.tsx

```diff
@@ -156,8 +156,6 @@ export class ScalingInput extends React.Component<ScalingInputProps, ScalingInpu
                     return `${width}px`;
                 }
                 return `${textLengthThreshold}ch`;
-            default:
```

Comment:
Is this fixing something? I noticed that sometimes the text in the input jumps around a bit, is this related?

Reply:
This is related to @steveklebanoff's comment that we shouldn't be always setting a default case for switch statements. I've not noticed the text jumping around.